Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -478,6 +478,14 @@ void ProgramShaderCache::Shutdown()
 				continue;
 			}
 
+			GLint link_status, delete_status;
+			glGetProgramiv(entry.second.shader.glprogid, GL_LINK_STATUS, &link_status);
```

Comment:
There's still more error checking to be done here.
If glGetProgramiv() fails it may not touch the output value. So either all these values should be initialized with fail-safe defaults or glGetError() should be called after each glGetProgramiv().